### PR TITLE
RF-14140: integration tests on tomcat

### PR DESCRIPTION
**Pull request by jstefek**

### CODE REVIEW

File: build/build-resources/src/main/java/org/richfaces/deployment/BaseDeployment.java [hunk 1]
```diff
@@ -247,9 +247,6 @@ private BaseDeployment withServletContainerSetup() {
 
         addMavenDependency("org.jboss.weld.servlet:weld-servlet");
 
-        addMavenDependency("org.jboss.el:jboss-el");
-        excludeMavenDependency("el-api");
-
```
Comment (ppitonak) [inline build/build-resources/src/main/java/org/richfaces/deployment/BaseDeployment.java]: why was it removed?

File: build/build-resources/src/main/java/org/richfaces/deployment/BaseDeployment.java [hunk 2]
```diff
@@ -261,23 +258,6 @@ public WebAppDescriptor apply(WebAppDescriptor webXml) {
                     .createListener()
                     .listenerClass("org.jboss.weld.environment.servlet.Listener");
 
-                // setup ExpressionFactory of JBoss EL (supports unified EL)
```
Comment (ppitonak) [inline build/build-resources/src/main/java/org/richfaces/deployment/BaseDeployment.java]: why was this removed?

File: build/build-resources/src/main/java/org/richfaces/deployment/BaseDeployment.java (hunk 1 above)
Comment (jstefek) [inline build/build-resources/src/main/java/org/richfaces/deployment/BaseDeployment.java]: see the JIRA for more details
